language_detector/languages.py

```diff
@@ -26,6 +26,18 @@
             'einen', 'kann', 'dem'
         ]
     },
-    #  include the English language here
-    # HINT: https://en.wikipedia.org/wiki/Most_common_words_in_English
+    {
+        'name': 'English',
+        'common_words': [            
+            'the', 'be', 'to', 'of', 'and', 'a', 'in', 'that', 'have',
+            'I', 'it', 'for', 'not', 'on', 'with', 'he', 'as', 'you', 
+            'do', 'at', 'this', 'but', 'his', 'by', 'from', 'they', 'we', 
+            'say', 'her', 'she', 'or', 'an', 'will', 'my', 'one', 'all', 
+            'would', 'there', 'their', 'what', 'so', 'up', 'out', 'if', 
+            'about', 'who', 'get', 'which', 'go', 'me', 'when', 'make', 
+            'can', 'like', 'time', 'no', 'just', 'him', 'know', 'take', 
+
+        ]
+    },
+    
```

Comment:
yeah we would fix that. thanks. i guess there won't be need to send a new pull request though, after we make these changes.